src/core_init_atmosphere/mpas_init_atm_gwd.F

```diff
@@ -117,15 +140,16 @@ function compute_gwd_fields(domain) result(iErr)
       character(len=StrKIND) :: geog_sub_path
       character(len=StrKIND+1) :: geog_data_path      ! same as config_geog_data_path, but guaranteed to have a trailing slash
 
+      TYPE(mpas_gwd_tile_type), POINTER :: tilesHead ! Pointer to linked list of tiles
```

Comment:
For consistency with the rest of MPAS, I'd suggest not capitalizing Fortran keywords.

Reply:
done.